frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -135,6 +135,11 @@ export const TableField = ({
     return description
   }, [fields.length, schema.addMoreRows, schema.maximumRows])
 
+  // If a table is the last field, Chrome and MS Edge sometimes truncate the table in print mode.
+  // We set the height of the table explicitly (depending on the number of cols and rows)
+  // for use in the media query so that the browser is able to render the full table in print mode
+  const printTableHeight = schema.columns.length * rows.length * 7.25
```

Comment:
Should we not hardcode the height? Will it work if we use the exact height of the table itself? We could set a ref on the table and get its current height. Or is 7.25rem a product of experimentation?

Reply:
yeah the thing is it's not the table height as it appears on the screen, cos when we print it becomes one column
e.g. 
<img width="863" alt="image" src="https://user-images.githubusercontent.com/56983748/208803771-48822b08-7770-4fc0-b36f-3495570fdc47.png">
becomes
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/56983748/208803837-4c8a31ef-b7ee-4dd8-8f17-6e9fb6efd448.png">

7.25rem was partially a product of experimentation - i added up the height of each cell + padding + margins, and gave a bit more buffer